Snakefile

```diff
@@ -455,27 +481,42 @@ rule force_tcp:
         cpus=1,
         mem_mb=1 * MEM_PER_CPU,
     params:
-        tp_p = lambda w: get_manual_tp_p(w.case, w.somatic)
+        tp=(
+            lambda w: config["models"][w.case][w.somatic]["tp"]
+            if "models" in config
+            and w.case in config["models"]
+            and w.somatic in config["models"][w.case]
+            else "NA"
+        ),
+        ploidy=(
+            lambda w: config["models"][w.case][w.somatic]["ploidy"]
+            if "models" in config
+            and w.case in config["models"]
+            and w.somatic in config["models"][w.case]
+            else "NA"
+        ),
```

Comment:
Hi @lculibrk,
I'm less sure this should have been in this PR right now.

Based on the types of suggestions from the snakemake --lint, once again.

Reply:
Do you mean that this rule/functionality should be omitted for a future version? I had added it for a collaborator's analysis and pushed it since it seemed to work.

Reply:
nope.  I meant that _I_ should not be messing with this many things in one pull request.

Reply:
Should be fine. Makes no practical changes unless different values specified. We'll need something like this anyways, for running a default of diploid, 100%.